File: swagger/models/get_client_services_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .client_service import ClientService

class GetClientServicesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		client_services
		Contains information about client pricing options.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'ClientServices': ['client_services', ClientService],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'client_services': 'ClientServices',
		}

	pagination_response = None
	client_services = None

File: swagger/Models/GetClientServicesResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetClientServicesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about client pricing options.
	 * 
	 * @var ClientService[] 
	 */
	public $clientServices;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'ClientServices' => ['clientServices', 'array', ClientService::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'clientServices' => ['ClientServices', 'array', ClientService::class],
			];
	}
}
